src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -241,3 +242,16 @@ export const hasOtherEvidence = formData =>
   _.get(DATA_PATHS.hasAdditionalDocuments, formData, false);
 export const hasPrivateEvidence = formData =>
   _.get(DATA_PATHS.hasPrivateEvidence, formData, false);
+
+export const servedAfter911 = formData =>
```

Comment:
This isn't _super_ computation-heavy, but I'd feel better if it were in a selector because I think this `depends` callback gets run rather a lot, if I remember right.

Reply:
Memoized!